ert3_examples/polynomial/experiments/evaluation/ensemble.yml

```diff
@@ -4,6 +4,7 @@ input:
   -
     source: stochastic.coefficients
     record: coefficients
+    mime: application/json
```

Comment:
?

Reply:
Hmm, this should not have gotten in there.